### ISSUE TITLE
Uploading a .gz mysql dump and nothing happens

### ISSUE THREAD

**Issue by zsoltbalogh** [djavorszky/ddn#194]
When I'm trying to import a database dump, I don't see the database on the list of databases.

The file I was uploading is this:
https://files.liferay.com/private/lrdcom/customer_lportal-2017-09-04.sql.gz

**Comment by djavorszky** [comment]
I see the following log entry:

```
2017/09/05 12:23:11 Failed creating file: open /ddn/web/dumps/customer_lportal-2017-09-04.sql.gz: no such file or directory
```

Quick question, did you upload the file or use the server browser?

**Comment by djavorszky** [comment]
Issue was a missing folder. I've corrected the Dockerfile in 4b2df69ada9b077d60e68874c0386bacea52e0a0

Also pushed to live, it should work now @zsoltbalogh.

Thanks for catching this!

**Comment by zsoltbalogh** [comment]
Thank you, it worked!
